autoload/gitgutter/hunk.vim

```diff
@@ -55,15 +55,19 @@ function! gitgutter#hunk#next_hunk(count) abort
   endif
 
   let current_line = line('.')
+  let hunk_total = len(hunks)
+  let hunk_number = 1
   let hunk_count = 0
   for hunk in hunks
     if hunk[2] > current_line
       let hunk_count += 1
       if hunk_count == a:count
         execute 'normal!' hunk[2] . 'Gzv'
+        redraw | echo printf('hunk %d of %d', hunk_number, hunk_total)
```

Comment:
Please could capitalise the first letter, i.e. "Hunk ..."

Reply:
Okay :cat: